demo/main.js

```diff
@@ -353,9 +350,7 @@ shakaDemo.Main = class {
       // Don't add the close button if in noInput mode; it doesn't make much
       // sense to stop playing a video if you can't start playing other videos.
 
-      // Register custom controls to the UI.
-      const closeFactory = new shakaDemo.CloseButton.Factory();
-      shaka.ui.Controls.registerElement('close', closeFactory);
+      // Register custom controls to the UI here
```

Comment:
If we no longer have any custom controls, we don't really need a comment here anymore.

Reply:
done